src/DssSpell.t.sol

```diff
@@ -340,4 +266,15 @@ contract DssSpellTest is DssSpellTestBase {
         }
         assertEq(expectedHash, actualHash);
     }
+
+    function testAAVEDirectBarChange() public {
```

Comment:
should probably move this to the "// Insert custom tests here" part.

Reply:
done at f0e678f021501cf3e4a5ffb51b6eca1396c8d5c8